documentatie/flowcharts/data-entry-state.md

```diff
@@ -5,21 +5,26 @@ The transition labels describe the endpoint that is used for performing the tran
 The "save" endpoint, used to for [First/Second]EntryInProgress states is kept out, because Mermaid doesn't render self-loops too well.
 
 ```mermaid
-stateDiagram
-FirstEntryNotStarted --> FirstEntryInProgress: claim
-#FirstEntryInProgress --> FirstEntryInProgress: claim
-#FirstEntryInProgress --> FirstEntryInProgress: save
-FirstEntryInProgress --> SecondEntryNotStarted: finalise
-FirstEntryInProgress --> FirstEntryNotStarted: delete
-SecondEntryNotStarted --> SecondEntryInProgress: claim
-#SecondEntryInProgress --> SecondEntryInProgress: claim
-#SecondEntryInProgress --> SecondEntryInProgress: save
-state is_equal <<choice>>
-is_equal --> Definitive: equal? yes
-is_equal --> EntriesDifferent: equal? no
-SecondEntryInProgress --> is_equal: finalise
-SecondEntryInProgress --> SecondEntryNotStarted: delete
-#EntriesDifferent --> EntriesDifferent: save
-# Will be Implemented in #130: EntriesDifferent --> NotStarted: delete
-EntriesDifferent --> Definitive: resolve
+stateDiagram-v2
+  [*] --> FirstEntryNotStarted
+  FirstEntryNotStarted --> FirstEntryInProgress: claim
+  #FirstEntryInProgress --> FirstEntryInProgress: save
+  FirstEntryInProgress --> SecondEntryNotStarted: finalise
+  FirstEntryInProgress --> FirstEntryNotStarted: delete
```

Comment:
'discard' or 'abort' would be a name that aligns a bit better with the language used in the user interface (applies to line 18 as well).

Reply:
Good suggestion, changed in 8d7a376ac46a13d6fabc78dbdd48f00a2481c19d